ProjectStructureAndRoles.md

```diff
@@ -6,9 +6,13 @@
+Each Sub Project is documented in one lead repository for the API definition and the API documentation, and optionally in an additional repository for each reference implementation (API code of the transformation function Northbound --> Southbound) for that API. The reference implementation repositories have the same name as the lead repository but with suffix _-RI<x>_.
 
-Each Sub Project must have a README.MD file (with a description of the Sub Project), a CODEOWNERS and a MAINTAINERS.MD file. A Sub Project should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the Sub Project. Here, the responsibilities must be clearly agreed upon between all of the Maintainers. This includes coordinating who is responsible for which issues and pull requests. The Contributors to a Sub Project are recorded by the mechanisms of GitHub in the Sub Project’s repository. Each Contributor, Codeowner and Maintainer should also be listed in the [PARTICIPANTS.MD](./PARTICIPANTS.MD) file in the Governance repository root directory. Each Sub Project should have a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and subdirectories /documentation/API_documentation, /documentation/MeetingMinutes, /documentation/SupportingDocuments, /code/API_code and /code/API_definitions. Writing permission to a Sub Project only codeowners should have.
+Each Sub Project lead repository must have a README.MD file (with a description of the Sub Project), a CODEOWNERS and a MAINTAINERS.MD file. A Sub Project (lead repository) should have at least one Maintainer. Ideally a Sub Project is managed by two or more Maintainers, depending on the size and scope of the Sub Project. Here, the responsibilities must be clearly agreed upon between all of the Maintainers. This includes coordinating who is responsible for which issues and pull requests. The Contributors to a Sub Project are recorded by the mechanisms of GitHub in the Sub Project’s repositories. Each Contributor, Codeowner and Maintainer should also be listed in the [PARTICIPANTS.MD](./PARTICIPANTS.MD) file in the Governance repository root directory. Each Sub Project lead repository should have a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and subdirectories /documentation/API_documentation, /documentation/MeetingMinutes, /documentation/SupportingDocuments and /code/API_definitions. Writing permission to a Sub Project lead repository only codeowners should have.
+  
+A reference implementation repository only must have a CODEOWNERS file, a license file, a GOVERNANCE.MD file (pointing to the Governance repository) and a subdirectory /code/API_code. Writing permission to a Sub Project reference implementation repository only codeowners should have.
+
+For ease of use there are templates for both types of repositories containing all necessary files which can be cloned and adapted.
```

Comment:
Maybe a reference to template files may help. Or something like templates could be found in CAMARA root repository.

Reply:
I've created template repos with all necessary file templates:
* https://github.com/camaraproject/Template_Lead_Repository
* https://github.com/camaraproject/Template_RI_Repository

Reply:
Yes I found them ... but a reference to these repositories in the doc may help to find them quickly.